iati/core/resources/test_data/202/valid_unique.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0"?>
 
 <root_element>
-  <iati-identifier>This is a unique identifier</iati-identifier><!-- iati-identifier does not replicate -->
+  <iati-identifier>This is a unique identifier</iati-identifier><!-- elements do not replicate -->
+  <other_unique_element></other_unique_element>
```

Comment:
This is testing only elements - what about attributes?

(presume this is a wider thing - both elements and attributes should be tested both positively and negatively for each type of rule since they can have different interfaces)